doc/api/process.md

```diff
@@ -708,6 +708,16 @@ console.log(process.env.TEST);
 // => undefined
 ```
 
+Note that, on Windows operating systems, environment variables are case-insensitive.
```

Comment:
Minor nit: Remove the `Note that, ` and just make it `On Windows, environment variables are case-insensitive.`